simple_salesforce-0.74.2.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
simple_salesforce-0.74.2.dist-info/METADATA,sha256=QwnESKh0_XdbASHqDrxnyVf7E_YNC8tMJ5Qrc0tlsFs,13951
simple_salesforce-0.74.2.dist-info/RECORD,,
simple_salesforce-0.74.2.dist-info/WHEEL,sha256=J3CsTk7Mf2JNUyhImI-mjX-fmI4oDjyiXgWT4qgZiCE,110
simple_salesforce-0.74.2.dist-info/top_level.txt,sha256=Hp1fBgmvtDXI6cwYWbFI72MRAeisTCrFHBY9uWsN1Kk,18
simple_salesforce/__init__.py,sha256=EFqMzaPXfh6chmTaMidlKB7db245LxLXlrXp74nnm6I,538
simple_salesforce/__pycache__/__init__.cpython-37.pyc,,
simple_salesforce/__pycache__/__version__.cpython-37.pyc,,
simple_salesforce/__pycache__/api.cpython-37.pyc,,
simple_salesforce/__pycache__/bulk.cpython-37.pyc,,
simple_salesforce/__pycache__/exceptions.cpython-37.pyc,,
simple_salesforce/__pycache__/login.cpython-37.pyc,,
simple_salesforce/__pycache__/util.cpython-37.pyc,,
simple_salesforce/__version__.py,sha256=fCZ_2hXDBWpTdEuujZ_bVee49RlW1qX9N8oymJXCs2I,510
simple_salesforce/api.py,sha256=A3MTMvjLFPqjiEywnUgNaxLoD12ZCQlDj2Pn6HXewj0,32017
simple_salesforce/bulk.py,sha256=kCyd3aYgbQ2yakfDjswvZ0ylip9D6EdRbRM4fF3u4lg,9108
simple_salesforce/exceptions.py,sha256=iFLU_n_KSb9r5mZkCADvNQlLTyVDZy51bAdsTHHX9Gk,3406
simple_salesforce/login.py,sha256=9gj6gm76IQrThqw8ce-RYkzBi4QMsr-v8UMd-7CZ-G8,7563
simple_salesforce/util.py,sha256=Ur7grJowssrJqChx2JYlZbdgvE4ByQMO6DWIeCX062Y,2515
